site/_includes/partials/deprecations-table.njk

```diff
@@ -0,0 +1,36 @@
+{% from 'macros/icon.njk' import icon with context %}
+<filtered-table>
+  <div class="deprecations-calendar__rows">
+    <div class="deprecations-calendar__row filtered__empty" hidden>
```

Comment:
The `filtered__empty` selector here is weird. In BEM methodology this suggests there's a parent element (a block) `filter`, but we only have the filtered-table component.